etc/widget.xml

```diff
@@ -30,6 +30,9 @@
                     <option name="small_badge_long" value="small_badge_long">
                         <label translate="true">Small service reviews badge long</label>
                     </option>
+                    <option name="sticky_badge" value="sticky_badge">
```

Comment:
I suggest to name it in more specific way `sticky_service_badge`, becuae maybe later we will have similar widgets for product reviews